tasks/configure.yml

```diff
@@ -17,4 +17,12 @@
       line: 'SOLR_JAVA_MEM="-Xms{{ solr_xms }} -Xmx{{ solr_xmx }}"'
     - regexp: "^SOLR_PORT="
       line: 'SOLR_PORT="{{ solr_port }}"'
-  notify: restart solr
\ No newline at end of file
+  notify: restart solr
+
+- name: Apply Solr timezone configuration changes.
```

Comment:
Instead of adding a separate task, could you just add one new `regexp/line` pair to the task above?